src/version-check.js

```diff
@@ -0,0 +1,25 @@
+var fs = require('fs')
+var chalk = require('chalk')
+var latestVersion = require('latest-version')
+
+module.exports = function () {
+  latestVersion('dat').then(function (version) {
+    fs.writeFile('../.dat-version', version, function (err) {
+      if (err) {
+        console.error(err)
+      }
+    })
+  })
+
+  if (fs.existsSync('../.dat-version')) {
```

Comment:
Let's avoid sync functions in this one so we don't block anything else. You can use `fs.stat` instead.